pom.xml

```diff
@@ -1,31 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright © 2017 CDAP
```

Comment:
2019

Reply:
thats because we did not have this license before. The plugin exists from 2017

Reply:
but you still need to update it now to 2018-2019, right?